pytr/main.py

```diff
@@ -206,6 +206,24 @@ def formatter(prog):
         help=info,
         description=info,
     )
+
+    # transactions
+    info = "Export all transactions directly from the API to a file (JSON)"
+    transactions = parser_cmd.add_parser(
+        "transactions",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        parents=[parser_login_args],
+        help=info,
+        description=info,
+    )
+    transactions.add_argument("output", help="Output path of JSON file", metavar="OUTPUT", type=Path)
```

Comment:
```suggestion
    transactions.add_argument(
        "output",
        help="Output path of JSON file [default: - (stdout)]",
        metavar="OUTPUT",
        type=argparse.FileType("w"),
        default="-",
        nargs="?",
    )
```

This would make it so if you give no arguments, you'd write to stdout. `TransactionsJsonExporter` would need to be updated to take a file-like object instead of a Path.